package.json

```diff
@@ -23,7 +23,7 @@
     "lint:fix": "lerna run lint:fix",
     "lint": "lerna run lint",
     "postinstall": "lerna bootstrap --hoist",
-    "release": "lerna publish --npm-tag prerelease",
+    "release": "npm run build && lerna publish --npm-tag prerelease",
```

Comment:
Shouldn't it be something like this?

- Release: Bump the version
- Prepublish hook: Run build command
- Publish: Upload to NPM

Reply:
Yep, it seems to be something related to lerna, forget about my comment